Chore: Remove wasm files from repo (#243)
yoctoproject/vscode-bitbake: README-DEVELOPER.md

To install the dependencies:
``` sh
npm install

> npm run build:wasm

Reviewer: I expect this will need to be explicitly run on some additional CI workflows?

Author: I think this is already here: https://github.com/yoctoproject/vscode-bitbake/blob/98c125d843a4c0156ae372203751e6ef03a26451/.github/workflows/main.yml#L66

Is that what you mean?

Reviewer: Yes!

See [the individual integration tests README](integration-tests/README.md).

## Tree-sitter
This extension uses [tree-sitter-bitbake](https://github.com/tree-sitter-grammars/tree-sitter-bitbake) and [tree-sitter-bash](https://github.com/tree-sitter/tree-sitter-bash) to parse the documents. When installing the project's dependencies with `npm install`, these tools are automatically built into WASM files. The versions of tree-sitter-bitbake and tree-sitter-bash are documented in [server/tree-sitter-bitbake.info](server/tree-sitter-bitbake.info) and [server/tree-sitter-bash.info](server/tree-sitter-bash.info) respectively, along with the version of the tree-sitter-cli that has been used.

> To update the .info files to the latest versions of tree-sitter-bitbake and tree-sitter-bash, it is recommended to use the scripts [scripts/update-tree-sitter-bitbake-wasm.sh](scripts/update-tree-sitter-bitbake-wasm.sh) and [scripts/update-tree-sitter-bash-wasm.sh](scripts/update-tree-sitter-bash-wasm.sh). After updating the .info files, it is required to use [scripts/build-tree-sitter-wasm.sh](scripts/build-tree-sitter-wasm.sh) in order to rebuild the WASM files.

Reviewer: You can explicit that there is a CI job doing that automatically and point to the .github-action file

Author: done

Reviewer: You'll need to update the URLs to match @WilsonZiweiWang latest PR https://github.com/yoctoproject/vscode-bitbake/pull/246#pullrequestreview-2130750927

Reviewer 2: done

See [the individual integration tests README](integration-tests/README.md).

## Tree-sitter

> This extension uses [tree-sitter-bitbake](https://github.com/tree-sitter-grammars/tree-sitter-bitbake) and [tree-sitter-bash](https://github.com/tree-sitter/tree-sitter-bash) to parse the documents. When installing the project's dependencies with `npm install`, these tools are automatically built into WASM files. The versions of tree-sitter-bitbake and tree-sitter-bash are documented in [server/tree-sitter-bitbake.info](server/tree-sitter-bitbake.info) and [server/tree-sitter-bash.info](server/tree-sitter-bash.info) respectively, along with the version of the tree-sitter-cli that has been used.

Reviewer 3: ```suggestion
This extension uses [tree-sitter-bitbake](https://github.com/tree-sitter-grammars/tree-sitter-bitbake) and [tree-sitter-bash](https://github.com/tree-sitter/tree-sitter-bash) to parse the documents. After installing the project's dependencies with `npm install`, these tools will build the WASM files required by `server/src/tree-sitter/parser.ts`. The versions of tree-sitter-bitbake and tree-sitter-bash are documented in [server/tree-sitter-bitbake.info](server/tree-sitter-bitbake.info) and [server/tree-sitter-bash.info](server/tree-sitter-bash.info) respectively, along with the version of the tree-sitter-cli that has been used.
```

> To update the .info files to the latest versions of tree-sitter-bitbake and tree-sitter-bash, it is recommended to use the scripts [scripts/update-tree-sitter-bitbake-wasm.sh](scripts/update-tree-sitter-bitbake-wasm.sh) and [scripts/update-tree-sitter-bash-wasm.sh](scripts/update-tree-sitter-bash-wasm.sh). The GitHub workflow [update-tree-sitter-wasm-file.yml](.github/workflows/update-tree-sitter-wasm-file.yml) is already responsible for doing it automatically.

Reviewer 3: ```suggestion
To update the .info files with the latest versions of tree-sitter-bitbake and tree-sitter-bash, it is recommended to use the scripts [scripts/update-tree-sitter-bitbake-wasm.sh](scripts/update-tree-sitter-bitbake-wasm.sh) and [scripts/update-tree-sitter-bash-wasm.sh](scripts/update-tree-sitter-bash-wasm.sh). The GitHub workflow [update-tree-sitter-wasm-file.yml](.github/workflows/update-tree-sitter-wasm-file.yml) is already responsible for doing it automatically.
```

Author: done
